docs/root/intro/version_history.rst

```diff
@@ -15,6 +15,11 @@ Version history
 * logger: all :ref:`logging levels <operations_admin_interface_logging>` can be configured
   at run-time: trace debug info warning error critical.
 * logger: added the ability to optionally set the log format via the :option:`--log-format` option.
+* sockets: added `IP_FREEBIND` socket option support for :ref:`listeners
```

Comment:
As long as you are in here and have to make a format change, can you move your round robin change to under "load balancer" (or whatever we use below)?

Reply:
I will do this in https://github.com/envoyproxy/data-plane-api/pull/579, where I'm also adding a load balancer entry.